participants/thaosle.sql

```diff
@@ -0,0 +1,19 @@
+# Question 1
+-- The youngest pet is Anna. (birthday is 2024-10)
+    -- Commands:
+        -- SELECT *
+        -- FROM pet
+        -- ORDER BY ASC
+        -- LIMIT 1
+# Question 2
+-- Yes, you would just have to query the records that contains a conditional
+-- that would have filter out any birthdays that happened 3-4 years ago
+--  (there should be 2 records)
+    -- Commands:
+        -- SELECT uuid
+        -- FROM pet
+        -- WHERE dateOfBirth > '2019-08-20' AND dateOfBirth < '2024-10-12'
```

Comment:
Hm... Time is an elusive force. Maybe there's a more abstract and future-proof way to go about this?

One solution would be working with https://dev.mysql.com/doc/refman/8.4/en/date-and-time-functions.html#function_date-sub